src/libConsensus/ConsensusLeader.h

```diff
@@ -80,7 +79,6 @@ class ConsensusLeader : public ConsensusCommon {
   std::map<unsigned int, bytes> m_commitFailureMap;
 
   // Tracking data for each consensus subset
-  // TODO: the vectors should be replaced by more space efficient DS
```

Comment:
This was more of an issue back when we could not forecast the final size of a shard.  At 600 nodes (or even a few hundred more beyond), the use of vectors here should be ok, given that the size of the data and the lifetime of a consensus are both not very large.